examples/oauth2/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  gateway:
+    image: apicast-test:latest
```

Comment:
think about whether you want to point to a (potentially, constantly changing) "latest", or a known numbered version that you know worked when you wrote the example...